components/common/hooks/useGetUsage.ts

```diff
@@ -1,38 +1,58 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {useMemo} from 'react';
-import {useSelector} from 'react-redux';
-
-import {FileSizes} from 'utils/file_utils';
+import {useEffect, useState} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
 
 import {CloudUsage} from '@mattermost/types/cloud';
+import {cloudFreeEnabled} from 'mattermost-redux/selectors/entities/preferences';
+import {getUsage} from 'mattermost-redux/selectors/entities/usage';
+import {
+    getMessagesUsage,
+    getFilesUsage,
+    getIntegrationsUsage,
+    getBoardsUsage,
+} from 'actions/cloud';
 
 // TODO: Replace this with actual usages stored in redux,
 // that ideally are updated with a websocket event in near real time.
```

Comment:
We can probably remove this comment. On the websocket event, are we updating this usage with the event of:

- Posts: Given that the posts is a rounded number, we proably don't need to update this count live for every post. 
- Plugin: We can listen for the enable plugin event that already exists
- Boards ?
- Files: Probably also doesn't need to be live update

Reply:
Do we have this websocket update covered in a ticket ?

Reply:
Perfect, wasn't aware Michael has it covered - Thanks!